Specs/Number/Spanish/NumberRangeModel.json

```diff
@@ -1,1283 +1,1206 @@
 [
-  {
```

Comment:
Format of this file is changed, so it's really hard to review the changes.
Could you please keep the original format and commit again?
Thanks.

Reply:
Already fixed the format of this file. Would you please take a review again. Thanks~

Reply:
Approved, waiting for the build.